packages/itwinui-css/backstop/tests/menu.html

```diff
@@ -1892,6 +1893,400 @@ <h3>Borderless button</h3>
+          <li
+            class="iui-menu-item"
+            role="menuitem"
+            tabindex="0"
+          >
+            <span class="iui-content">Option 2D</span>
+          </li>
+          <li
+            class="iui-menu-item"
+            role="menuitem"
+            tabindex="0"
+          >
+            <span class="iui-content">Option 2E</span>
+          </li>
+          <li
+            class="iui-menu-item iui-active"
+            role="menuitem"
+            aria-checked="true"
+            tabindex="0"
+          >
+            <span class="iui-content">Option 2F</span>
+            <svg-checkmark
+              class="iui-icon"
+              aria-hidden="true"
+            ></svg-checkmark>
+          </li>
+          <li
+            class="iui-menu-item iui-disabled"
+            role="menuitem"
+            tabindex="0"
+          >
+            <span class="iui-content">Option 2G</span>
+          </li>
+          <li
+            class="iui-menu-item"
+            role="menuitem"
+            tabindex="0"
+          >
+            <span class="iui-content">Option 2H</span>
+          </li>
+          <li
+            class="iui-menu-item"
+            role="menuitem"
+            tabindex="0"
+          >
+            <span class="iui-content">Option 2I</span>
+          </li>
+          <li
+            class="iui-menu-item"
+            role="menuitem"
+            tabindex="0"
+          >
+            <span class="iui-content">Option 2J</span>
+          </li>
+          <li
+            class="iui-menu-item iui-disabled"
+            role="menuitem"
+            tabindex="0"
+          >
+            <span class="iui-content">Option 2K</span>
+          </li>
+          <li
+            class="iui-menu-item"
+            role="menuitem"
+            tabindex="0"
+          >
+            <span class="iui-content">Option 2L</span>
+          </li>
+          <li
+            class="iui-menu-item"
+            role="menuitem"
+            tabindex="0"
+          >
+            <span class="iui-content">Option 2M</span>
+          </li>
+          <li
+            class="iui-menu-item iui-active"
+            role="menuitem"
+            tabindex="0"
+          >
+            <span class="iui-content">Option 2N</span>
+            <svg-checkmark
+              class="iui-icon"
+              aria-hidden="true"
+            ></svg-checkmark>
+          </li>
+          <li
+            class="iui-menu-item"
+            role="menuitem"
+            tabindex="0"
+          >
+            <span class="iui-content">Option 2O</span>
+          </li>
+        </ul>
+      </div>
+
+      <br />
+
+      <h3 style="padding-bottom: 12px">Flat</h3>
+
+      <ul
+        class="iui-menu iui-flat"
+        role="menu"
+      >
+        <li
+          class="iui-menu-item"
+          role="menuitem"
+          tabindex="0"
+          id="test-multilevel"
+        >
+          <span class="iui-content">Option 1</span>
```

Comment:
nit: Should this have a subheading too? Because the unequal heights of the menu items seems a bit off.

Reply:
I added a subheading here and updated the examples around it.